spring-boot-hello-world/src/main/java/com/jalasoft/convert/model/ocrconvert/OCR.java

```diff
@@ -35,8 +35,8 @@ public void convertOCR(String imageInputName, String lang) throws TesseractExcep
             System.out.println("Test from image is: ");
```

Comment:
you should no longer use these system.print, delete them or replace them with the logger

Reply:
hello @TolabaLib the issue is now solved, thanks for the observation